content/docs/rfcs/60/README.md

```diff
@@ -0,0 +1,70 @@
+---
+slug: 60
+title: 60/STATUS-URL-SCHEME
+name: Status URL Scheme
+status: draft
+category: Standards Track
+tags: waku-application
+editor: Felicio Mununga <felicio@status.im>
+contributors:
+---
+
+# Abstract
+
+This document describes URL scheme for previewing and deep linking content as well as for triggering actions.
+
+# Background / Rationale / Motivation
+
+## Requirements
+
+### Related scope
+
+#### Features
+
+- Onboarding website
+- Link preview
+- Link sharing
+- Deep linking
+- Routing and navigation
+- Payment requests
+- Chat creation
+
+# Wire Format Specification / Syntax
+
+## Schemes
+
+- Internal `status-app://`
+- External `https://` (i.e. univers/deep links)
+
+## Paths
+
+| Name | Url | Description |
+| ----- | ---- | ---- |
+| User profile | `/u/<data>#<signature>` | Preview/Open user profile |
+| | `/u#<compressed_user_public_key>` | |
+| | `/u#<ens_name>` | |
+| Community | `/c/<data>#<signature>` | Preview/Open community |
+| | `/c#<compressed_community_public_key>` | |
+| Community channel | `/cc/<data>#<signature>`| Preview/Open community channel |
+| | `/cc/<channel_uuid>#<compressed_community_public_key>` | |
```

Comment:
```suggestion
| Name | Url | Description |
| ----- | ---- | ---- |
| User profile | `/u/<encoded_data>#<encoded_signature_and_user_chat_key>` | Preview/Open user profile |
| | `/u#<user_chat_key>` | |
| | `/u#<ens_name>` | |
| Community | `/c/< encoded_data >#<encoded_signature_and_community_chat_key>` | Preview/Open community |
| | `/c#<community_chat_key>` | |
| Community channel | `/cc/< encoded_data >#< encoded_signature_and_community_chat_key >`| Preview/Open community channel |
| | `/cc/<channel_uuid>#<community_chat_key>` | |
```

Reply:
`<encoded_signature_and_community_chat_key>` instead of `<signature>`

done in

https://github.com/status-im/status-web/pull/407/commits/d6300069b9a5694946b91aaf6e64198b367d03f7#diff-67f802f9ab57f928747d93bf45b7dbc9062af282d8e39e496c4d4ad5ad97dd0c

because

https://github.com/status-im/status-web/pull/407/commits/f0e59b379817b477db68972a98c17109bd952436#diff-4c64b597af17789d552f8acb35dfc18def8c8c09dbff90f2548d91357ee272df

reported in

https://github.com/paulmillr/noble-secp256k1/issues/43#issuecomment-1020214968

Reply:
But if other implementations can prove this change is redundant, please let me know asap.

Reply:
Reverting at https://github.com/vacp2p/rfc/pull/602/files/0b29aa5794ad603b6afc027ab8d3d39a11a4720c#r1213035855